cowprotocol/accounting/rewards/mainnet/full_bonding_pools_query_4056263.sql

```diff
@@ -3,19 +3,29 @@
 with
 full_bonding_pools as (
     select
-        from_hex('0x8353713b6D2F728Ed763a04B886B16aAD2b16eBD') as pool_address,
+        from_hex('0x8353713b6D2F728Ed763a04B886B16aAD2b16eBD') as pool_address, -- deprecated
```

Comment:
Should we rename this to deprecated? Having Gnosis vs Gnosis DAO seems a bit awkwardly implicit